packages/data-management/src/AddTrackDrawerWidget/components/AddTrackDrawerWidget.js

```diff
@@ -82,9 +82,13 @@ function AddTrackDrawerWidget({ model }) {
       return
     }
     trackAdapter.features = trackData.config
+    const trackId = `${trackName
+      .toLowerCase()
```

Comment:
reason why Date.now is added?

Reply:
`trackId`s have to be globally unique, but track names don't. This way if someone names a track something generic like "variants", and then does the same thing in another dataset, the `trackId`s won't clash. Or if they add a track, rename it, and then add a second track with the same name as the first track's original name, it won't cause a duplicate identifier. I figured appending `Date.now()` was an easy way to keep some readability of `trackId`s but still have it be unique.

Reply:
It's just that this can result in non-deterministic trackId, which is something that the connection code worked on with object-hash to make them deterministic.

Reply:
If that is not a concern that's ok, it was just something that seemed important in other circumstances like the connection code

Reply:
Yeah, I think in this case since it's a normal track config and doesn't have the limitations of a connection track config, it's better to be non-deterministic.